queue/queue.py

```diff
@@ -28,3 +28,22 @@ def len(self):
 
   def len(self):
     pass
+
+
+class Linked_List:
+	def __init__(self):
+		self.head = None
+		self.tail = None
+
+ 	def change_head(self, node=None):
+		if self.head == None:
+			self.head = node
+			self.tail = node
+		else:
+			temp = self.head.get_next()
```

Comment:
What if the head does not have a next pointer and there is only one value in the queue?